include/nntile/kernel/rope.hh

```diff
@@ -0,0 +1,29 @@
+/*! @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+ *                              (Skoltech), Russia. All rights reserved.
+ *                 2023-present Artificial Intelligence Research Institute
+ *                              (AIRI), Russia. All rights reserved.
+ *
+ * NNTile is software framework for fast training of big neural networks on
+ * distributed-memory heterogeneous systems based on StarPU runtime system.
+ *
+ * @file include/nntile/kernel/rope.hh
+ * Rotary Positional Embedding
+ *
+ * @version 1.0.0
+ * */
+
+#pragma once
+
+#include <nntile/kernel/rope/cpu.hh>
+#include <nntile/defs.h>
+// #ifdef NNTILE_USE_CUDA
+// #include <nntile/kernel/rope/cuda.hh>
+// #endif // NNTILE_USE_CUDA
```

Comment:
Deadcode.

Reply:
All of these dead codes will be reenabled soon, as we are developing CUDA kernel. It works now with CPU code.